copyleaks/models/submit/properties/pdf.py

```diff
@@ -74,3 +75,37 @@ def set_rtl(self, value):
         assert value
 
         self.rtl = value
+
+
```

Comment:
Missing create pdf option.
look at the documentation at properties.pdf.create

Reply:
add to the example file this lines after this code "file_submission.set_properties(scan_properties)" :

#pdf = Pdf() # Creating instance of Pdf.
#pdf.set_create(True) # Setting the create pdf to True to generate PDF report.
#scan_properties.set_pdf(pdf)  # Will generate PDF report.